website/src/components/footer.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Footer = () => <footer>
+  <span className='footer-text'>© 2020-2024 Björn Büttner and contributors.</span>
+  <ul id="policy-list">
+    <li>
+      <a href="#" >About</a>
+    </li>
+    <li>
+      <a href="#" >Privacy Policy</a>
+    </li>
+    <li>
+      <a href="#" >Licensing</a>
+    </li>
+    <li>
+      <a href="/imprint" >Imprint</a>
```

Comment:
please remove the unneeded space in the a tag

Reply:
Done